src/time_tagger/measurement/dao.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+import TimeTagger as TT
+from shared.constants.constants import INTEGRATION_TIME
+from time_tagger.builder import TimeTaggerBuilder
+
+
+@dataclass
+class CountRateMeasurementReqParams:
+    channels: list[int]
+    time_tagger_network_proxy: object
+
+
+class MeasurementDao:
+    def __init__(self, time_tagger_builder: TimeTaggerBuilder) -> None:
+        self.time_tagger_builder = time_tagger_builder
```

Comment:
is this `time_tagger_builder ` being used by the class?

Reply:
It is not, I forgot to remove this injection :)